waku/options.go

```diff
@@ -0,0 +1,13 @@
+package waku
+
+import "github.com/waku-org/waku-go-bindings/waku/common"
+
+type WakuNodeOption func(*WakuNode)
+
+// This allows you to pass arbitrary valid config options to Nwaku baed on https://github.com/waku-org/nwaku/blob/master/waku/factory/external_config.nim
```

Comment:
Typo
```suggestion
// This allows you to pass arbitrary valid config options to Nwaku based on https://github.com/waku-org/nwaku/blob/master/waku/factory/external_config.nim
```

Something to bear in mind is that not all the `external_config.nim` params are suitable for libwaku. For example, the "REST" params are not handled in libwaku.

Reply:
As I mention, the extraOptions is expected to disappear once libwaku API stabilizes. I would treat it as a debug/dev thing:)